airflow/dag_processing/manager.py

```diff
@@ -526,6 +526,9 @@ def deactivate_stale_dags(
         dags_parsed = session.execute(query)
 
         for dag in dags_parsed:
+            if not Path(dag.fileloc).is_relative_to(dag_directory):
```

Comment:
Just want to confirm if `dag.fileloc` no longer exists, we're removing them as well. I guess it's handled elsewhere?

Reply:
Yes we are deleting the DAGs here - https://github.com/apache/airflow/blob/9af26368df3651b21c66ccefa6147158ecf2a8d7/airflow/dag_processing/manager.py#L552